### PR TITLE
Participant model refractor 85

### PR DESCRIPTION
**Pull request by KTryCode**
Zwinąłem atrybuty dotyczące ParticipantData do odpowiedniej klasy.
Nie podoba mi się jedynie dość długa metoda `from(ParticipantDTO participantDTO` w klasie `ParticipantCreator`, ale na to na "późniejszy" refractor :D

### CODE REVIEW

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/ParticipantCreator.java [hunk 1]
```diff
@@ -1,5 +1,9 @@
 package pl.oaza.warszawa.dor.rekolekcje.api.participants.domain;
 
+import static pl.oaza.warszawa.dor.rekolekcje.api.participants.domain.DateConverter.convertToDateTime;
+
+import java.util.Set;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/ParticipantCreator.java:5]: \[Checkstyle\] WARNING: Import statement for 'java\.util\.Set' is in the wrong order\. Should be in the 'STANDARD\_JAVA\_PACKAGE' group, expecting group 'THIRD\_PARTY\_PACKAGE' on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/ParticipantCreator.java [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+import java.util.stream.Collectors;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/ParticipantCreator.java:6]: \[Checkstyle\] WARNING: Import statement for 'java\.util\.stream\.Collectors' is in the wrong order\. Should be in the 'STANDARD\_JAVA\_PACKAGE' group, expecting group 'THIRD\_PARTY\_PACKAGE' on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/PersonalData.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
+package pl.oaza.warszawa.dor.rekolekcje.api.participants.domain;
+
+import java.time.LocalDateTime;
+import javax.persistence.Embeddable;
+import javax.persistence.Embedded;
+import lombok.AllArgsConstructor;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/PersonalData.java:6]: \[Checkstyle\] WARNING: Import statement for 'lombok\.AllArgsConstructor' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/PersonalData.java [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+import lombok.Builder;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/PersonalData.java:7]: \[Checkstyle\] WARNING: Import statement for 'lombok\.Builder' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/PersonalData.java [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+import lombok.NoArgsConstructor;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/PersonalData.java:8]: \[Checkstyle\] WARNING: Import statement for 'lombok\.NoArgsConstructor' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/PersonalData.java [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+import pl.oaza.warszawa.dor.rekolekcje.api.participants.value.AddressValue;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/PersonalData.java:9]: \[Checkstyle\] WARNING: Import statement for 'pl\.oaza\.warszawa\.dor\.rekolekcje\.api\.participants\.value\.AddressValue' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/PersonalData.java [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+import pl.oaza.warszawa.dor.rekolekcje.api.participants.value.PersonalDataValue;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/PersonalData.java:10]: \[Checkstyle\] WARNING: Import statement for 'pl\.oaza\.warszawa\.dor\.rekolekcje\.api\.participants\.value\.PersonalDataValue' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/parish/domain/Parish.java [hunk 8]
```diff
@@ -1,14 +1,13 @@
...
-import lombok.AllArgsConstructor;
-import lombok.Builder;
-import lombok.NoArgsConstructor;
-import pl.oaza.warszawa.dor.rekolekcje.api.parish.dto.ParishDTO;
-
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import lombok.AllArgsConstructor;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/parish/domain/Parish.java:7]: \[Checkstyle\] WARNING: Import statement for 'lombok\.AllArgsConstructor' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/parish/domain/Parish.java [hunk 9]
```diff
@@ -1,14 +1,13 @@
...
+import lombok.Builder;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/parish/domain/Parish.java:8]: \[Checkstyle\] WARNING: Import statement for 'lombok\.Builder' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/parish/domain/Parish.java [hunk 10]
```diff
@@ -1,14 +1,13 @@
...
+import lombok.NoArgsConstructor;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/parish/domain/Parish.java:9]: \[Checkstyle\] WARNING: Import statement for 'lombok\.NoArgsConstructor' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/parish/domain/Parish.java [hunk 11]
```diff
@@ -1,14 +1,13 @@
...
+import pl.oaza.warszawa.dor.rekolekcje.api.parish.dto.ParishDTO;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/parish/domain/Parish.java:10]: \[Checkstyle\] WARNING: Import statement for 'pl\.oaza\.warszawa\.dor\.rekolekcje\.api\.parish\.dto\.ParishDTO' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/parish/domain/ParishService.java [hunk 12]
```diff
@@ -1,12 +1,11 @@
...
 
-import pl.oaza.warszawa.dor.rekolekcje.api.parish.dto.ParishDTO;
+import static java.util.Objects.requireNonNull;
 
 import java.util.Arrays;
 import java.util.List;
 import java.util.stream.Collectors;
-
-import static java.util.Objects.requireNonNull;
+import pl.oaza.warszawa.dor.rekolekcje.api.parish.dto.ParishDTO;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/parish/domain/ParishService.java:8]: \[Checkstyle\] WARNING: 'pl\.oaza\.warszawa\.dor\.rekolekcje\.api\.parish\.dto\.ParishDTO' should be separated from previous import group by one line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/Experience.java [hunk 13]
```diff
@@ -1,22 +1,21 @@
 package pl.oaza.warszawa.dor.rekolekcje.api.participants.domain;
 
+import static pl.oaza.warszawa.dor.rekolekcje.api.participants.domain.DateConverter.convertToUtc;
+
 import com.google.common.collect.Sets;
+import java.time.LocalDateTime;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/Experience.java:6]: \[Checkstyle\] WARNING: Import statement for 'java\.time\.LocalDateTime' is in the wrong order\. Should be in the 'STANDARD\_JAVA\_PACKAGE' group, expecting group 'THIRD\_PARTY\_PACKAGE' on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/Experience.java [hunk 14]
```diff
@@ -1,22 +1,21 @@
...
+import java.util.Set;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/Experience.java:7]: \[Checkstyle\] WARNING: Import statement for 'java\.util\.Set' is in the wrong order\. Should be in the 'STANDARD\_JAVA\_PACKAGE' group, expecting group 'THIRD\_PARTY\_PACKAGE' on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/Experience.java [hunk 15]
```diff
@@ -1,22 +1,21 @@
...
+import java.util.stream.Collectors;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/Experience.java:8]: \[Checkstyle\] WARNING: Import statement for 'java\.util\.stream\.Collectors' is in the wrong order\. Should be in the 'STANDARD\_JAVA\_PACKAGE' group, expecting group 'THIRD\_PARTY\_PACKAGE' on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/Experience.java [hunk 16]
```diff
@@ -1,22 +1,21 @@
...
+import javax.persistence.CascadeType;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/Experience.java:9]: \[Checkstyle\] WARNING: Import statement for 'javax\.persistence\.CascadeType' is in the wrong order\. Should be in the 'STANDARD\_JAVA\_PACKAGE' group, expecting group 'THIRD\_PARTY\_PACKAGE' on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/Experience.java [hunk 17]
```diff
@@ -1,22 +1,21 @@
...
+import javax.persistence.Embeddable;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/Experience.java:10]: \[Checkstyle\] WARNING: Import statement for 'javax\.persistence\.Embeddable' is in the wrong order\. Should be in the 'STANDARD\_JAVA\_PACKAGE' group, expecting group 'THIRD\_PARTY\_PACKAGE' on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/Experience.java [hunk 18]
```diff
@@ -1,22 +1,21 @@
...
+import javax.persistence.OneToMany;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/Experience.java:11]: \[Checkstyle\] WARNING: Import statement for 'javax\.persistence\.OneToMany' is in the wrong order\. Should be in the 'STANDARD\_JAVA\_PACKAGE' group, expecting group 'THIRD\_PARTY\_PACKAGE' on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/RetreatTurn.java [hunk 19]
```diff
@@ -1,18 +1,17 @@
...
-import pl.oaza.warszawa.dor.rekolekcje.api.participants.value.RetreatTurnValue;
-
 import javax.persistence.Entity;
 import javax.persistence.FetchType;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
 import javax.persistence.JoinColumn;
 import javax.persistence.ManyToOne;
+import lombok.AllArgsConstructor;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/RetreatTurn.java:10]: \[Checkstyle\] WARNING: Import statement for 'lombok\.AllArgsConstructor' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/RetreatTurn.java [hunk 20]
```diff
@@ -1,18 +1,17 @@
...
+import lombok.Builder;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/RetreatTurn.java:11]: \[Checkstyle\] WARNING: Import statement for 'lombok\.Builder' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/RetreatTurn.java [hunk 21]
```diff
@@ -1,18 +1,17 @@
...
+import lombok.NoArgsConstructor;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/RetreatTurn.java:12]: \[Checkstyle\] WARNING: Import statement for 'lombok\.NoArgsConstructor' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/RetreatTurn.java [hunk 22]
```diff
@@ -1,18 +1,17 @@
...
+import lombok.Setter;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/RetreatTurn.java:13]: \[Checkstyle\] WARNING: Import statement for 'lombok\.Setter' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/RetreatTurn.java [hunk 23]
```diff
@@ -1,18 +1,17 @@
...
+import pl.oaza.warszawa.dor.rekolekcje.api.participants.value.RetreatTurnValue;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/RetreatTurn.java:14]: \[Checkstyle\] WARNING: Import statement for 'pl\.oaza\.warszawa\.dor\.rekolekcje\.api\.participants\.value\.RetreatTurnValue' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/value/ExperienceValue.java [hunk 24]
```diff
@@ -1,10 +1,9 @@
 package pl.oaza.warszawa.dor.rekolekcje.api.participants.value;
 
-import lombok.Builder;
-import lombok.Value;
-
 import java.time.ZonedDateTime;
 import java.util.Set;
+import lombok.Builder;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/value/ExperienceValue.java:5]: \[Checkstyle\] WARNING: Import statement for 'lombok\.Builder' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/value/ExperienceValue.java [hunk 25]
```diff
@@ -1,10 +1,9 @@
...
+import lombok.Value;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/value/ExperienceValue.java:6]: \[Checkstyle\] WARNING: Import statement for 'lombok\.Value' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/jwt/JwtAuthenticationEntryPoint.java [hunk 26]
```diff
@@ -1,13 +1,12 @@
...
-import org.springframework.security.core.AuthenticationException;
-import org.springframework.security.web.AuthenticationEntryPoint;
-import org.springframework.stereotype.Component;
-
+import java.io.IOException;
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import java.io.IOException;
+import org.springframework.security.core.AuthenticationException;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/jwt/JwtAuthenticationEntryPoint.java:7]: \[Checkstyle\] WARNING: Import statement for 'org\.springframework\.security\.core\.AuthenticationException' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/jwt/JwtAuthenticationEntryPoint.java [hunk 27]
```diff
@@ -1,13 +1,12 @@
...
+import org.springframework.security.web.AuthenticationEntryPoint;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/jwt/JwtAuthenticationEntryPoint.java:8]: \[Checkstyle\] WARNING: Import statement for 'org\.springframework\.security\.web\.AuthenticationEntryPoint' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/jwt/JwtAuthenticationEntryPoint.java [hunk 28]
```diff
@@ -1,13 +1,12 @@
...
+import org.springframework.stereotype.Component;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/jwt/JwtAuthenticationEntryPoint.java:9]: \[Checkstyle\] WARNING: Import statement for 'org\.springframework\.stereotype\.Component' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/jwt/JwtTokenUtil.java [hunk 29]
```diff
@@ -4,16 +4,15 @@
 import io.jsonwebtoken.Claims;
 import io.jsonwebtoken.Jwts;
 import io.jsonwebtoken.SignatureAlgorithm;
+import java.util.Date;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/jwt/JwtTokenUtil.java:7]: \[Checkstyle\] WARNING: 'java\.util\.Date' should be separated from previous import group by one line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/jwt/JwtUser.java [hunk 30]
```diff
@@ -1,13 +1,12 @@
 package pl.oaza.warszawa.dor.rekolekcje.api.security.jwt;
 
 import com.fasterxml.jackson.annotation.JsonIgnore;
+import java.util.Collection;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/jwt/JwtUser.java:4]: \[Checkstyle\] WARNING: 'java\.util\.Collection' should be separated from previous import group by one line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/users/Authority.java [hunk 31]
```diff
@@ -16,7 +12,10 @@
 import javax.persistence.ManyToMany;
 import javax.persistence.Table;
 import javax.validation.constraints.NotNull;
-import java.util.List;
+import lombok.AllArgsConstructor;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/users/Authority.java:15]: \[Checkstyle\] WARNING: Import statement for 'lombok\.AllArgsConstructor' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/users/Authority.java [hunk 32]
```diff
@@ -16,7 +12,10 @@
...
+import lombok.Builder;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/users/Authority.java:16]: \[Checkstyle\] WARNING: Import statement for 'lombok\.Builder' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/users/Authority.java [hunk 33]
```diff
@@ -16,7 +12,10 @@
...
+import lombok.Getter;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/users/Authority.java:17]: \[Checkstyle\] WARNING: Import statement for 'lombok\.Getter' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/users/Authority.java [hunk 34]
```diff
@@ -16,7 +12,10 @@
...
+import lombok.NoArgsConstructor;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/users/Authority.java:18]: \[Checkstyle\] WARNING: Import statement for 'lombok\.NoArgsConstructor' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/users/User.java [hunk 35]
```diff
@@ -20,8 +16,11 @@
 import javax.persistence.TemporalType;
 import javax.validation.constraints.NotNull;
 import javax.validation.constraints.Size;
-import java.util.Date;
-import java.util.List;
+import lombok.AllArgsConstructor;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/users/User.java:19]: \[Checkstyle\] WARNING: Import statement for 'lombok\.AllArgsConstructor' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/users/User.java [hunk 36]
```diff
@@ -20,8 +16,11 @@
...
+import lombok.Builder;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/users/User.java:20]: \[Checkstyle\] WARNING: Import statement for 'lombok\.Builder' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/users/User.java [hunk 37]
```diff
@@ -20,8 +16,11 @@
...
+import lombok.Getter;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/users/User.java:21]: \[Checkstyle\] WARNING: Import statement for 'lombok\.Getter' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/users/User.java [hunk 38]
```diff
@@ -20,8 +16,11 @@
...
+import lombok.NoArgsConstructor;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/users/User.java:22]: \[Checkstyle\] WARNING: Import statement for 'lombok\.NoArgsConstructor' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.

File: rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/users/User.java [hunk 39]
```diff
@@ -20,8 +16,11 @@
...
+import lombok.Setter;
```
Comment (sputnikci) [inline rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/users/User.java:23]: \[Checkstyle\] WARNING: Import statement for 'lombok\.Setter' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.

**Review by qiubix — APPROVED** [review]

**Pull request merged**